repos/tap/components/feature/featureEditor.js

```diff
@@ -1,12 +1,34 @@
-import React from 'react'
+import React, { useCallback, useState } from 'react'
 import { AceEditor } from 'SVComponents/aceEditor'
+import { useFeature } from 'SVHooks/useFeature'
+import { noOp, noOpObj } from '@keg-hub/jsutils'
+/**
+ * 
+ * @param {Object} props 
+ * @param {Object} props.activeFile
+ * @param {Object} props.onChange
+ */
+export const FeatureEditor = props => {
+  const {
+    activeFile=noOpObj,
+    onChange=noOp
+  } = props
+  
+  const { feature } = useFeature({path: activeFile?.fullPath}) || {}
+  const [activeFeat, setActiveFeat] = useState(feature)
 
+  const onFeatureEdit = useCallback((text) => {
+    text !== feature.content &&
+      !text.trim() &&
+      setFeature({ ...feature, text })
```

Comment:
Pretty sure this should be `content`, not `text`
* And if it's not it will need to be updated, cause all test files should have the property content
* If that's not the case right now, don't worry about it, but I though I already added this change

Reply:
* I didnt' really make any updates on the `onFeatureEdit` callback, I just moved it, but I'll update `text` to `content`
* Updated fontSize on sidebar node item
* Added hover styles on sidebar node items